src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard</title>
+</head>
+<body>
+  <header>
+    <h1>LeaderBoard</h1>
+  </header>
+  <section class="main-section">
+    <div class="recent-scores">
+      <div class="heading-area">
+        <h2>Recent Scores</h2>
+        <button class="button" type="button">Refresh</button>
+      </div>
+      <table class="table">
+
+      </table> 
+    </div>
+    <div class="add-score">
+        <div class="heading-area">
+          <h2>Add your Score</h2>
+        </div>
+        <div class="form">
+          <input id="scorename" type="text" placeholder="Your name" required><br>
+          <input id="yourscore" type="text" placeholder="Your score" required><br>
```

Comment:
- [ ] [OPTIONAL] It could be better to change your `score` input in `type="number"`, you'll have something like `<input id="yourscore" type="number" placeholder="Your score" required>` . Kindly fix it 😃